source/geod24/LocalRest.d

```diff
@@ -1093,28 +1062,28 @@ unittest
     auto node1 = factory("normal", 1);
     auto node2 = factory("byzantine", 2);
 
-    static void testFunc(geod24.concurrency.Tid parent)
-    {
-        auto node1 = factory("this does not matter", 1);
-        auto node2 = factory("neither does this", 2);
-        assert(node1.pubkey() == 42);
-        assert(node1.last() == "pubkey");
-        assert(node2.pubkey() == 0);
-        assert(node2.last() == "pubkey");
-
-        node1.recv(42, Json.init);
-        assert(node1.last() == "recv@2");
-        node1.recv(Json.init);
-        assert(node1.last() == "recv@1");
-        assert(node2.last() == "pubkey");
-        node1.ctrl.shutdown();
-        node2.ctrl.shutdown();
-        geod24.concurrency.send(parent, 42);
-    }
+    auto node12 = factory("this does not matter", 1);
+    auto node22 = factory("neither does this", 2);
+
+    assert(node12.pubkey() == 42);
+    assert(node12.last() == "pubkey");
+    assert(node22.pubkey() == 0);
+    assert(node22.last() == "pubkey");
 
-    auto testerFiber = geod24.concurrency.spawn(&testFunc, geod24.concurrency.thisTid);
```

Comment:
This changes the test quite a bit. Originally those nodes were spawned from a non-main thread, but not anymore. Why?